src/configs/webpack/root/module/rules/30-fonts.js

```diff
@@ -1,17 +1,5 @@
 export default rules => [
     ...rules,
-    {
-        test: /\.svg$/,
-        use: {
-            loader: require.resolve('url-loader'),
-            options: {
```

Comment:
Why the options not needed anymore?

Reply:
Good point. I forgot about that. Restored.